frontend/src/shared/modules/filters/components/filterTypes/MultiSelectAsyncFilter.vue

```diff
@@ -33,6 +33,12 @@
               <span class="el-checkbox__inner" />
             </span>
           </div>
+          <template v-if="config.id === 'organizations'">
```

Comment:
Instead of validating `config.id === 'organizations'`, could you validate `option.logo`

Reply:
is there a reason why `option.logo` is better? asking cause some organizations don't have logo, so I'd have to do a double check `option.logo || option.logo === null`

Reply:
You're right, my mistake, this is ok 😄